pkg/component/networking/vpn/shoot/shoot.go

```diff
@@ -607,6 +616,30 @@ func (v *vpnShoot) container(secrets []vpnSecret, index *int) *corev1.Container
 	}
 }
 
+func (v *vpnShoot) tunnelControllerContainer() *corev1.Container {
+	return &corev1.Container{
+		Name:            "tunnel-controller",
+		Image:           v.values.Image,
+		ImagePullPolicy: corev1.PullIfNotPresent,
+		Command:         []string{"/bin/tunnel-controller"},
```

Comment:
Is this needed?

Reply:
Yes, the `tunnel-controller` binary is part of the `vpn-client` image with a different `ENTRYPOINT`.